### ISSUE TITLE
Find area of segmentation

### ISSUE THREAD

**Issue by albertboston** [facebookresearch/maskrcnn-benchmark#61]
## ❓ Questions and Help

Similar to how the original [Mask_RCNN repo](https://github.com/matterport/Mask_RCNN/blob/master/samples/coco/inspect_model.ipynb) shows methods of displaying segmentation only, and not the classification bounding box, is there any way to extract the segmentation as a numpy array? My ultimate goal for a project is to find the number of pixels in each segmentation, and this would be extremely helpful!

Thanks so much.

**Comment by fmassa** [comment]
Yes, there is definitely a way of getting the segmentation mask as a numpy array. Have a look at `demo/predictor.py` to see how I extracted the predictions (a `BoxList` object with several attributes, including `masks`) into an image, and how you can do the operations as you want in there.

Let me know if you have more questions, I'm typing from my phone so it's hard to get direct code pointers.
